src/components/schema-editor-sidebar/SchemaEditorSidebar.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { SidebarCard, Tabs } from '..';
+import { IconAssetsImage, IconLink, IconTypeface } from '../..';
+import { SidebarTitle } from '../sidebar-title';
+import { TabPanel } from '../tabs/Tabs';
+import { SSchemaEditorSidebar } from './styles/SSchemaEditorSidebar';
+import { SSidebarCategoryWrapper } from './styles/SSidebarCategoryWrapper';
+
+export const SchemaEditorSidebar: React.FC = () => {
+  return (
+    <SSchemaEditorSidebar>
+      <Tabs initialValue={0}>
+        <TabPanel title="All Fields" style={{ padding: "1.25rem 1rem" }}>
+          <SSidebarCategoryWrapper>
```

Comment:
Having this implementation won't make the component usable. This is an example use case, but this component should be the abstraction. CategoryWrapper has to be it's own component, and then be instantiated by the actual implementation.